chroot/run.go

```diff
@@ -1004,10 +1005,18 @@ func setupChrootBindMounts(spec *specs.Spec, bundlePath string) (undoBinds func(
 	var fs unix.Statfs_t
 	removes := []string{}
```

Comment:
Looks like we append to removes, but never use it?  I'm not sure why it's being picked up now and not before this commit.

```
./tests/tools/build/golangci-lint run

chroot/run.go:1279:3: ineffectual assignment to `removes` (ineffassign)

		removes = append(removes, roEmptyDir)
```